PYCS_Language/Services/TextParser.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SplitAndMerge
+{
+    public class TextParser : ITextParser
+    {
+        public void ParseText(string[] text)
+        {
+            BasicOperationsHandler handler = new BasicOperationsHandler();
+            List<string> result = new List<string>();
+
+            foreach (string s in text)
+            {
+                List<string> words = new List<string>(s.Split(new char[] { ' ' }));
+
+                for (int i = 0; i < words.Count; i++)
+                {
+                    if (words[i] == Constants.PLUS)
+                    {
+                        result.Add(handler.AddOperation(Convert.ToInt32(words[i - 1]), Convert.ToInt32(words[i + 1])));
+                    }
+                    if (words[i] == Constants.MINUS)
```

Comment:
Please use "if...else if "to avoid useless checks
OR use pattern matching here

Reply:
Done!